### PR TITLE
change assert to exception in python code

### PR DESCRIPTION
**Pull request by boolafish**
fix issue #57

### CODE REVIEW

File: plasma/child_chain/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class TxAlreadySpentException(Exception):
+    """the transaction is already spent"""
+
+
+class InvalidTxSignatureException(Exception):
+    """the signature is invalid"""
+
+
+class TxInputAndOutputAmountMismatchException(Exception):
```
Comment (kfichter) [inline plasma/child_chain/exceptions.py]: Requesting this be changed to `TxAmountMismatchException` for readability.

File: plasma/child_chain/exceptions.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    """tx input total amount is not equal to output total amount"""
+
+
+class UnAuthorizedBlockSubmissionException(Exception):
```
Comment (kfichter) [inline plasma/child_chain/exceptions.py]: Requesting this be changed to `InvalidBlockSignatureException` for consistency.

**Comment by boolafish** [comment]
updated the PR with the comment feedback! : )

**Comment by kfichter** [comment]
@boolafish Awesome, approving these changes.

**Pull request merged**